pkgs/by-name/po/poetry/unwrapped.nix

```diff
@@ -45,14 +47,14 @@ buildPythonPackage rec {
     owner = "python-poetry";
     repo = "poetry";
     tag = version;
-    hash = "sha256-RpAoADxZmH9hQSEjufLBoKJsxIc74RnRxZB3RVNk/iE=";
+    hash = "sha256-2u0idmnXY0FNvc8peDMWIFAeGH/xdIfMrMErUdxc0UA=";
   };
 
   patches = [
     # https://github.com/python-poetry/poetry/pull/9939
     (fetchpatch2 {
-      url = "https://github.com/python-poetry/poetry/commit/89c0d02761229a8aa7ac5afcbc8935387bde4c5b.patch?full_index=1";
-      hash = "sha256-YuAevkmCSTGuFPfuKrJfcLUye1YGpnHSb9TFSW7F1SU=";
+      url = "https://github.com/python-poetry/poetry/pull/9939/commits/c2387ff3c878ab608d7616e4984fc01c4226416c.patch";
```

Comment:
Either use `?full_index=1` or `fetchpatch`.

Reply:
Can you point me on where to read what is the difference and why? These are new things to me 🙂

Reply:
But changed that.

Reply:
See https://github.com/NixOS/nixpkgs/issues/257446.